src/django/api/management/commands/resetdb.py

```diff
@@ -9,3 +12,11 @@ def handle(self, *args, **options):
         call_command('reset_schema', '--noinput')
         call_command('migrate')
         call_command('loadfixtures')
+
+        facility_one = Facility.objects.first()
```

Comment:
Consider moving this code (or something similar to `makefixtures`. `resetdb` was intended to just be a helper that runs 3 commands in sequence. Ensuring that we have test data that allows us to exercise the UI is more the responsibility of the fixture.

Reply:
Django querysets are lazy, so you could optionally use a slightly shorter version

```python
for f in Facility.objects.all().order_by('id')[:2]:
    f.location = Point(0, 0)
    f.save()
```

Reply:
Cool, I'll move this out of resetdb and into makefixtures.

Reply:
I ended up changing the mock facility creation code to:

```
def make_facility(facility_item):
    (created_at, updated_at) = make_created_updated()
    fields = facility_item['fields']
    # A workaround to have the CSV parser work on a single row
    for row in csv.reader([fields['raw_data']]):
        parsed = row

    pk = facility_item['pk']
    point_is_fixed = pk % 70 == 0

    return {
        'model': 'api.facility',
        'pk': pk,
        'fields': {
            'name': parsed[1],
            'address': parsed[2],
            'country_code': COUNTRY_CODES[parsed[0].upper()],
            'location': make_point(point_is_fixed),
            'created_from': pk,
            'created_at': created_at,
            'updated_at': updated_at,
        }
    }
```

and also changed `make_point` to make it return a point at 0 0 if the boolean True was passed in as an argument.